Packs/CommonTypes/IncidentFields/incidentfield-Threat_Family_Name.json

```diff
@@ -0,0 +1,31 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "threatfamilyname",
+    "closeForm": false,
+    "content": true,
+    "description": "Threat Family Name Associated with an Attacking Vector",
```

Comment:
Any type of example to provide? We want the user to understand when to use Threat Family Name VS Malware Family name